firmware/dev/h7dev/src/app/configs/App_SharedSignalConfig.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+struct H7DevWorld;
+#define World H7DevWorld
```

Comment:
should this have the exact same contents as firmware/dev/h7dev/src/app/configs/App_SharedStateMachineConfig.h?

Reply:
just removed the shared code that needed these via cmake instead